packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -179,5 +179,19 @@
     "event_properties",
     "user_properties"
   ],
-  "anyOf": [{ "required": ["user_id"] }, { "required": ["device_id"] }]
+  "anyOf": [{ "required": ["user_id"] }, { "required": ["device_id"] }],
+  "if": {
```

Comment:
This does seem to work as expected. The only strange thing is that if `event_type` is empty, then this "if-then" also seems to trigger and "language" becomes required. 

The validation still fails as expected, it's just that language also becomes required and errors if language isn't included in the event. The scenario is tested by test `it('errors - event_type required'...`.

If there's a better way of doing this, please let me know!